### PR TITLE
Fix translations

### PR DESCRIPTION
**Pull request by peyman-mohtashami**

### CODE REVIEW

**Comment by nivemaham** [comment]
@peyman-mohtashami Can you merge with `dev` branch again?

File: config.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version='1.0' encoding='utf-8'?>
-<widget android-versionCode="6" id="uk.ac.ucl.homeapp" ios-CFBundleIdentifier="uk.ac.ucl.homeapp" ios-CFBundleVersion="6" version="0.7.6" xmlns:android="http://schemas.android.com/apk/res/android">
+<widget android-versionCode="8" id="uk.ac.ucl.homeapp" ios-CFBundleIdentifier="uk.ac.ucl.homeapp" ios-CFBundleVersion="8" version="0.7.8" xmlns:android="http://schemas.android.com/apk/res/android">
```
Comment (nivemaham) [inline config.xml]: Bundle version 8 already exists for iOS due to the bug we had on ios during last release. Not sure if this would work.

**Comment by peyman-mohtashami** [comment]
I changed both code versions to 9 and version to 0.7.9

**Review by nivemaham — APPROVED** [review]
LGTM

**Pull request merged**